scipy-1.0/paper.tex

```diff
@@ -447,6 +447,14 @@ \section*{Project organisation and community}
 \textbf{Community beyond the SciPy library}
 
 \textbf{Maintainers and contributors}
+The SciPy project has approximately 15 active maintainers (people who review
+others' contributions, and in general do everything needed to ensure that
+the software and the project move forward).  It also has an additional ~100
+unique contributors for every 6-monthly release cycle.  Maintainers are
+critical to the health of the project\cite{eghbal2016}; they enable contributions
+from a much larger group, and their skills and efforts largely determine how
+fast the project moves forward.  Anyone with the interest and skills can become
+a maintainer; the SciPy Developer Guide provides guidance on how to do that.
```

Comment:
Citation?

Reply:
yeah, probably. I was thinking of going through the whole paper at the end and checking for consistency in citations